spec/operators/audit-spec.ts

```diff
@@ -168,13 +168,13 @@ describe('audit operator', () => {
     });
   });
 
-  it('should emit no values if duration is a never', () => {
+  it('should emit no values and never complete if duration is a never', () => {
     testScheduler.run(({ hot, cold, expectObservable, expectSubscriptions }) => {
       const e1 = hot('  ----abcdefabcdefabcdefabcdefa|');
       const e1subs = '  ^----------------------------!';
       const e2 = cold(' -');
-      const e2subs = '  ----^------------------------!';
-      const expected = '-----------------------------|';
+      const e2subs = '  ----^-------------------------';
+      const expected = '------------------------------';
```

Comment:
Hmmm... I'll need to think about this. I think this makes the most sense. Given that there are items to audit, and we're waiting for a duration that just never arrives... But I want a second opinion: @cartant

Reply:
Yeah, I think this makes sense, too. Above all, the behaviour needs to be consistent between operators and, AFAICT, `delayWhen` doesn't complete if its notifier is `NEVER`: https://github.com/ReactiveX/rxjs/blob/0d3094f29b5f6f2cc4d2829f8a0e20addd2106a3/spec/operators/delayWhen-spec.ts#L131-L144

So this LGTM.